src/trans-word-level/property.cpp

```diff
@@ -61,3 +62,128 @@ void property(
     prop_handles.push_back(std::move(handle));
   }
 }
+
+/*******************************************************************\
+
+Function: states_equal
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+exprt states_equal(
+  std::size_t i,
+  std::size_t k,
+  const std::vector<symbol_exprt> &state_vars)
+{
+  exprt::operandst conjuncts;
+  conjuncts.reserve(state_vars.size());
+
+  for(auto &var : state_vars)
+  {
+    auto i_var = timeframe_symbol(i, var);
+    auto k_var = timeframe_symbol(k, var);
+    conjuncts.push_back(equal_exprt(i_var, k_var));
+  }
+
+  return conjunction(std::move(conjuncts));
```

Comment:
`conjunction` takes a const reference, so the `std::move` won't help much I believe.

Reply:
I'll do a PR for `conjunction`.